packages/migrations/src/actions/2025.01.02T00-00-00.legacy-user-org-cleanup.ts

```diff
@@ -0,0 +1,31 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2024.12.02T00-00-00.legacy-user-org-cleanup.ts',
+  async run({ sql, connection }) {
+    // Delete all the organizations owned by a legacy user
```

Comment:
We need to make sure those are not active

Reply:
I checked for schema publishes and all of them were ~2 years old